### PR TITLE
Updates to the Readme that include mission

### PR DESCRIPTION
**Pull request by aronchick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+## The Kubeflow Mission
+
+Our goal is to help folks use ML more easily, by letting Kubernetes to do what it's great at:
+- Easy, repeatable, portable deployments
+- Deploying and managing loosely-coupled microservices
+- Scaling based on demand
+
+Because ML practitioners use so many different types of tools, it is a key goal that you can customize the stack to whatever your requirements (within reason), and let the system take care of the "boring stuff." While we have started with a narrow set of technologies, we are working with many different projects to include additional tooling.
+
+Ultimately, we want to have a single deployable artifact (a manifest) that gives you an easy to use ML stack _anywhere_ Kubernetes is already running and can self configure based on the cluster it deploys into.
```
Comment (foxish) [inline README.md]: a set of deployable manifests? Because we're doing a directory of them, with the application step being:
`kubectl apply -f dir/ -R

Comment (aronchick) [reply README.md]: great point, fixed!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
-# Kubeflow Anywhere
+# Kubeflow
 
-The Kubeflow project is a new open source Github repo dedicated to reducing friction in doing Machine Learning on Kubernetes. Contained in this repository are:
+The Kubeflow project is dedicated to making Machine Learning on Kubernetes easy, portable and scalable. Contained in this repository are:
```
Comment (jlewi) [inline README.md]: The elements listed below aren't included in the repository; they are included in the Kubeflow distribution. Should we change the text to something like
Contained in the Kubeflow distribution? or
Contained in the Kubeflow stack?

Comment (vishh) [reply README.md]: +1.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
 
-This document details the steps needed to run the kubeflow project in different environments, such as Minikube (local laptop). Google Kubernetes Engine, etc. 
+This document details the steps needed to run the kubeflow project in any environment in which Kubernetes runs.
+
+## The Kubeflow Mission
+
+Our goal is to help folks use ML more easily, by letting Kubernetes to do what it's great at:
+- Easy, repeatable, portable deployments
+- Deploying and managing loosely-coupled microservices
+- Scaling based on demand
```
Comment (vishh) [inline README.md]: add: managing diverse infrastructure

File: README.md [hunk 4]
```diff
@@ -17,23 +28,21 @@ In order to quickly setup all components of the stack, run:
...
+
 ### Minikube
 
 [Minikube](https://github.com/kubernetes/minikube) is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-
 node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day. 
-The below steps apply to a minikube cluster - the latest version as of writing this documetation is 0.23.0. You must also have 
-kubectl configured to access minikube.
+The below steps apply to a minikube cluster - the latest version as of writing this documetation is 0.23.0. You must also have kubectl configured to access minikube.
 
-### Google Kubernetes Engine
```
Comment (vishh) [inline README.md]: Do not delete this section. It is the best way currently to try out GPUs k8s clusters.

Comment (aronchick) [reply README.md]: That's fine, but it shouldn't be here - or if it is, it should have other solutions as well. As it stands, it makes it seem that this is the only way to create a production cluster.

**Comment by foxish** [comment]
@aronchick PTAL. Needs resolving conflict. Good to merge after that

**Comment by foxish** [comment]
Squashing and merging this now.

**Pull request merged**